selftests/functional/test_standalone.py

```diff
@@ -62,6 +62,13 @@ def test_failtest_nasty2(self):
         self.assertIn("Exception: Unable to get exception, check the traceback"
                       " for details.", result.stdout)
 
+    def test_failtest_nasty3(self):
```

Comment:
The name of the test is misleading, because we want to get a `TestError`. Maybe it should be called `errortest_nasty.py` ?